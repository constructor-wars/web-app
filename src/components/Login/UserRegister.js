import React from 'react';

function UserRegister(props) {
  return (
    <form>
      <input className=""placeholder={props.placeholder} />
    </form>
  );
}

export default UserRegister;
