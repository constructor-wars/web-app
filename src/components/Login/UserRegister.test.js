import React from "react";
import UserRegister from "./UserRegister";
import renderer from "react-test-renderer";

describe("UserLogin", () => {
  it("matches the snapshot", () => {
    const tree = renderer.create(<UserRegister />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
