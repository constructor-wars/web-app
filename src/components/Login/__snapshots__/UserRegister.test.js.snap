// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserLogin matches the snapshot 1`] = `
<form>
  <input
    className=""
  />
</form>
`;
